prow/config/plugins.yaml

```diff
@@ -118,6 +118,17 @@ blockades:
       This PR modifies the files under the docs folder and requires the docs team to follow up on the PR.
 
       /label docs
+  - repos:
+      - pingcap/tidb
+    blockregexps:
+      - ^sessionctx/variable/
+      - ^config/config.toml.example$
+    explanation: |
+      This PR modifies significant config files which require review from domain experts from anyone of
+      
+      @qw4990, @bb7133, @XuHuaiyu, @cfzjywxk, or @morgo
+      
+      You can request review from anyone of them by `/cc @reviewer`
```

Comment:
Since we already have a concrete list of domain experts to ask review for, maybe we can don't bother the author to manually type, just auto add the experts to review list?

Reply:
Maybe the current configuration is feasible because GitHub will send the mentioned notification to all domain experts when the PR is opened (because they are @) to remind them to pay attention to this PR. 

Sending a requested review in advance will cause unnecessary pending review.

When the other merge conditions of the PR are satisfied, the domain experts still have no response. The contributor or the committer can manually `/cc` request the review of the domain experts.

Reply:
I mean remove the @ part and directly add these experts to review list. So no double notification and maybe this entire explain message is not needed. Current message gives me a feeling that I should `\cc` immediately after the message is prompted and then double notification.

Reply:
@tisonkun, what do you think?

The blockade plugin doesn't support the auto-request review. Auto request review is usually the responsibility of the [blunderbuss](https://book.prow.tidb.io/#/plugins/blunderbuss) plugin. If it needs to be automatically allocated according to the file path, need to introduce the OWNERS file mechanism.

for example:
https://github.com/kubernetes/test-infra/blob/03e3720d1199cbd13462b82a5f545c544799ccd8/config/prow/OWNERS#L3

Reply:
@Mini256 I don't have a strong feeling on request review or not. But stuck at so-called upstream limitation is undesirable for tichi to handle the PR review process.

We cannot say because of k8s fault, we stuck. If so, think of a resolution to avoid such unreliable dependencies.

Reply:
@tisonkun, the blockade plugin can request all the domain experts by `/cc` command, but cannot select one of them to request.

@zhangyangyu, updated in the new commit.